_releases/2022-01-06-1.3.0-released.md

```diff
@@ -14,6 +14,30 @@ In this release we included [XXX PRs since the 1.2.2 release](https://github.com
 
 ## Interpreter
 
+The awaited Crystal interpreter has been [merged](https://github.com/crystal-lang/crystal/pull/11159). To use it, you need to compile Crystal with a special flag. The official releases (.deb, .rpm, docker images, etc.) are **not** being compiled with it, since its [missing substantial features](https://github.com/crystal-lang/crystal/issues/11555) for real use cases.
+
+The following is an extract of the [post](https://crystal-lang.org/2021/12/29/crystal-i.html) where we introduced this feature.
```

Comment:
I would skip that here and instead add `Read more about the interpreter in our blog post: [*Crystal's interpreter – A very special holiday present*](https://crystal-lang.org/2021/12/29/crystal-i.html)` at the end of this section.
```suggestion
```